src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1617,8 +1617,11 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public string ToUpper() { return default(string); }
         public string ToUpperInvariant() { return default(string); }
         public string Trim() { return default(string); }
+        public string Trim(char c) { return default(string); }
```

Comment:
I mentioned this in in https://github.com/dotnet/coreclr/pull/2041/files#r44751579, but mentioning it again here: Should the parameter be named `trimChar` to be consistent with the existing `trimChars`?